### PR TITLE
Updated Badge design system file

### PR DESCRIPTION
**Pull request by itz-deepak**
I have  updated the badge design system file.

### CODE REVIEW

File: Badges/badges.design-system.mdx [hunk 1]
```diff
@@ -1 +1,110 @@
...
+
+### Font
+
+For all buttons font is : **`font-medium`**
+
+### Border Radius 🔲
+
+The normal border-radius for the components are `rounded`.
+
+### Hover Effect :
```
Comment (Rajdip019) [inline Badges/badges.design-system.mdx:29]: Remove the hover part as there is no hover effect on badges.

**Comment by Rajdip019** [comment]
Add more details on what colors are used and what is the convention of that. Convention means like the color of bg is always used a shade of 100 and text when its dark is always used a shade 800 like that.

Add details about text that what text sizes are used like sm, md ... etc and where are they used.

And also what are the default paddings and margins.

Except that everything looks awesome. @itz-deepak

File: Badges/badges.design-system.mdx (hunk 1 above)
Comment (itz-deepak) [reply Badges/badges.design-system.mdx:29]: Hover effect is used in 2 html file of badges

**Comment by Rajdip019** [comment]
Tag me for review when you add all the changes. @itz-deepak

**Comment by itz-deepak** [comment]
Please review I have made the changes

**Comment by Rajdip019** [comment]
Looks perfect. Great work @itz-deepak 

Congrats 🎉, your PR got merged. To be eligible for the swags from ClueLess. Follow these steps.

1. Star this repository. ⭐
2. SignIn / SignUp in [[ClueLess Website](https://www.clueless.tech/)](https://www.clueless.tech/)
3. Go to the profile section from the top right-hand corner.



Comment your profile URL under this comment tagging the person who merged your PR.

That’s it. Follow the leaderboard!

And don’t forget to share your achievement with your friends.

**Pull request merged**